src/main/java/com/google/swarm/tokenization/common/DLPTransform.java

```diff
@@ -99,7 +99,7 @@ public void processElement(ProcessContext c) throws IOException {
           if (this.requestBuilder.build().getSerializedSize() > DLP_PAYLOAD_LIMIT) {
             String errorMessage =
                 String.format(
-                    "Payload Size %s Exceeded Batch Size %s",
+                    "DLPTransform:DLPInspect: Payload Size %s Exceeded Batch Size %s",
```

Comment:
Optional- Change the log.error to warn

Reply:
Yes got that. Will run this through the team and decide accordingly.